modules/os2forms_encrypt/src/Helper/FormHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Drupal\os2forms_encrypt\Helper;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form helper class.
+ */
+class FormHelper {
+
+  /**
+   * Removes 'element_encrypt' element from element forms.
+   *
+   * @param array $form
+   *   The form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param string $form_id
+   *   The form id.
+   */
+  public function formAlter(array &$form, FormStateInterface $form_state, string $form_id) {
+    if ('webform_ui_element_form' === $form_id) {
+      if (isset($form['element_encrypt'])) {
+        unset($form['element_encrypt']);
```

Comment:
Would it be better to set access = false

Reply:
With the current `preSave` hook ensuring encryption on all elements are enabled, the end behaviour would be the same.

Maybe from an understanding point of view using `access = false` makes more sense? I'm not sure.

Reply:
To avoid potential issues with other parts of code accessing this `element_encryption` we will use `access = false`.